src/DacpacTool/PackageDeployer.cs

```diff
@@ -92,12 +92,33 @@ public void Deploy(string targetDatabaseName)
             try
             {
                 var services = new DacServices(ConnectionStringBuilder.ConnectionString);
+                services.Message += HandleDacServicesMessage;
                 services.Deploy(Package, targetDatabaseName, true, DeployOptions);
                 _console.WriteLine($"Successfully deployed database '{targetDatabaseName}'");
             }
-            catch (Exception ex)
+            catch (DacServicesException ex)
```

Comment:
Other exception types?

Reply:
Uhm, yes, of course. Not sure what I was thinking ;). Added a catch for the generic Exception now.